variables.tf

```diff
@@ -72,6 +72,21 @@ variable "include_self_ingress_rule" {
 }
 
 variable "allow_major_version_upgrade" {
-  description = "hether or not to allow major version upgrades"
+  description = "Whether or not to allow major version upgrades"
   default = "no"
 }
+
+variable "auto_minor_version_upgrade" {
+  description = "Whether or not to enable auto minor version upgrades"
+  default = "no"
+}
+
+variable "storage_type" {
+  description = " (Optional) One of standard(magnetic), gp2(general purpose SSD)."
```

Comment:
No need to mark as "Optional" in the description here since the terraform registry automatically determines what is optional and what is required based on the absence or presence of defaults.

Propose changing to `"The type of storage to use, either \"standard\" (magnetic) or \"gp2\" (general purpose SSD)."`.

Reply:
I'll update this.